src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -2,192 +2,217 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
+using System;
+using System.ComponentModel;
 using System.IO;
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.SqlServer.Dac;
+using System.Reflection;
+using System.Collections.Generic;
 
 namespace Microsoft.SqlTools.ServiceLayer.DacFx.Contracts
 {
     /// <summary>
-    /// Class to define deployment options. 
+    /// Class to define deployment option default value and the description
+    /// Value: Default and selected value of the deployment options
+    /// Description: Description of the deployment options
+    /// DisplayName: To Diaply the options in ADS extensions UI in SchemaCompare/SQL-DB-Project/Dacpac extensions
+    /// </summary>
+    public class DeploymentOptionProperty<T>
+    {
+        public DeploymentOptionProperty(T value, string description = "", string displayName = "")
```

Comment:
Why do description and displayName need to have default values of empty string?

Reply:
Some DeploymentOptions like CommandTimeout/LongRunnigCommandTimeout are giving with a default value. Instead of passing a null values to the description and displayNames during that initialization for all other options, have added a default null value at here!